packages/codd/scripts/codd.py

```
@@ -64,8 +79,59 @@ def collect_files(root: Path, config: cc.CoddConfig) -> list[Path]:
+    """
+    if not config.code_include:
+        return []
+    included = _glob_relpaths(root, config.code_include)
+    excluded = _glob_relpaths(root, config.code_exclude)
+    return [root / rel for rel in sorted(included - excluded)]
+
+
+def _read_source_text(path: Path) -> str:
+    """ソースファイルをテキストとして読む。
+
+    Python ファイル（``.py``）は PEP 263 の宣言済みエンコーディング（先頭2行の
```

Ruff (0.15.21)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)